### DISCUSSION TITLE
Standup (Tuesday, May 12, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#57] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sent #56 
 - First try with environment variables

**What are your priorities for the next 24 hours?**:
 - Deploy app on heroku/netlify

**Blockers**:
 - Hard days at work

**Shoutouts**:
 - @DavBot02 for his commitment on the frontend

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing

**What are your priorities for the next 24 hours?**:
 - Work on form handling

**Blockers**:
 - 

**Shoutouts**:
 - @antonieto for being a delightful individual

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing

**What are your priorities for the next 24 hours?**:
 - work on documenting endpoints

**Blockers**:
 - primarily homework 

**Shoutouts**:
 - @DavBot02 @antonieto for been incredibly handsome and spent last night beside me
